ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/TransactionService.scala

```diff
@@ -136,6 +143,96 @@ class TransactionService(session: LedgerSession) extends LedgerTestSuite(session
       }
   }
 
+  test(
```

Comment:
No action to be taken here, as this is a very useful test to have, but I think we're adding a test that tests the Ledger API server, rather than the backing ledger. If this test framework is so helpful, we should probably consider spinning it off and use it for both internal integration tests and the external Ledger API test tool.

Reply:
Agreed. The reason why the test is spread across 3 participants is to check the privacy of the backing ledger. But since the ledger api server is actually doing the final blinding, we cannot tell the difference on the client side of the ledger api of whether the privacy aware split of the transaction happened already between ledger and participant, or only between participant and ledger api.

Reply:
And yes, I agree 💯 to also use this test framework for ledger api/sandbox only integration tests.